setup.py

```diff
@@ -8,6 +8,7 @@
   "aiohttp==3.10.11",
   "aiohttp_cors==0.7.0",
   "aiofiles==24.1.0",
+  "anyio==4.6.2",
```

Comment:
I don't see why we need to add another dependency. We should be able to do asynchronous file operations with the existing dependencies.